src/doc/ndarray_for_numpy_users/mod.rs

```diff
@@ -116,6 +117,17 @@
 //! </tr>
 //! </table>
 //!
+//! # The ndarray ecosystem
+//!
+//! `ndarray` does not provide advanced linear algebra routines out of the box (e.g. SVD decomposition).
+//! Most of the routines that you can find in `SciPy.linalg`/`NumPy.linalg` are provided by another crate,
```

Comment:
```suggestion
//! Most of the routines that you can find in `scipy.linalg`/`numpy.linalg` are provided by another crate,
```

The module names are lowercase.